packages/keyda-demo/src/lib/Input/index.js

```diff
@@ -24,6 +31,17 @@ const Input = (props) => {
   ];
 
   const keyDispatch = useKeyStateDispatch();
+  const keyState = useKeyStateState();
+  const inputRef = useRef();
+
+  useImperativeHandle(inputRef, () => ({
+    setLastKeyDown: (value) => {
+      setLastKeyDown(value);
+    },
+    setLastKeyUp: (value) => {
+      setLastKeyUp(value);
+    },
+  }));
```

Comment:
This is the part of adding a function to the `ref` of `input`.
This is the most confusing part while I work on this feature.
https://ko.reactjs.org/docs/hooks-reference.html#useimperativehandle